### DISCUSSION TITLE
Q&A about MIMO OFDM Transmissions over the CDL Channel Model

### DISCUSSION THREAD

**Discussion by ShelleyTimon** [NVlabs/sionna#57] [Q&A] [answered]
Hi everyone,
1. I'm trying to simulate the effect of mobility using MIMO_OFDM_Transmissions_over_the_CDL_Channel_Model. I simulate the uplink channel in frequency domain and increase the modulation order to 4 with other parameters unchanged. The result is shown as below.
![image](https://user-images.githubusercontent.com/42461121/194747080-d11e3d5b-e735-4853-b520-71c5459d1667.png)
I am a little confused why the BLER curve rise after it reaches the bottom instead of reaching the bottom and then remaining stable(just like the figure you show in example when modulation order is 2). I try to remove channel coding and find BLER is flat after its bottom. In this way, can I improve the LDPC decoding algorithm to produce a monotonous BLER figure or do I have to improve the precision of channel estimation to produce such a BLER curve when I increase modulation order?
2. Besides, if I want to extend CDL model to multiple transmitters scenario, how can I do this? Do I need to configure the locations of transmitters to calculate LOS angles according to TS 38.901 since current cdl class doesn't have such an interface?
Thanks a lot.

**Comment by jhoydis** [comment, marked as answer]
Hi,

1) When you increase the modulation order in this scenario, you operate the system at a higher rate than what it can support. Your BLER is essentially stable at an extremely high value of almost 50%. There are various things that you could do to make it work:
    - Decrease the code rate.
    - Use more pilot symbols. In the current example only the first OFDM symbol within a slot is used. This was done to make the effect of channel aging as strong as possible. If you use, e.g., ``"pilot_ofdm_symbol_indices" : [2, 11]``, you get the following result:
    
![image](https://user-images.githubusercontent.com/5190129/194814823-60d31d69-dec1-4ec3-a96c-3352e15929f5.png)




2) The CDL models are not supposed to be used for multiple transmitters or receivers. They are for MIMO point-to-point links only.
For multiuser simulations, you need to resort to the UMi, Uma, or RMa models, as for example done in this notebook: https://nvlabs.github.io/sionna/examples/Realistic_Multiuser_MIMO_Simulations.html.

**Reply by ShelleyTimon** [reply]
Many thanks!
